docker/pdo_client.bashrc

```diff
@@ -0,0 +1,5 @@
+/usr/bin/echo -e "\
+NOTE: to setup environment, \"source /project/pdo/tools/start_client.sh\"\n\
+   with the arguments \"--mode build\" to build a new client environment (default mode)\n\
+   or \"--mode copy\" to copy one from the xfer directory"
+
```

Comment:
put this in the tools directory.

Reply:
i wouldn't have seen this really as a tool and we still have to do dockerfile-action as the copying of tools is not enough?  But if you prefer i can put it there and then replace the current COPY with with RUN 'ln -s ...' command ...